src/instance_group.ts

```diff
@@ -132,15 +132,22 @@ export default class InstanceGroupManager {
         ctx: Context,
         type: string,
         region: string,
+        environment: string,
     ): Promise<Array<InstanceGroup>> {
         const groups = await this.getAllInstanceGroups(ctx);
 
         function byTypeAndRegion(group: InstanceGroup) {
```

Comment:
Please rename byTypeAndRegion to something similar to byTypeRegionAndEnv. The same for getAllInstanceGroupsByTypeAndRegion.

Reply:
Done